app/scripts/app.js

```diff
@@ -19,7 +19,9 @@ require.config({
         ejs: '../bower_components/ejs/ejs',
         moment: '../bower_components/moment/moment',
         moment_range: '../bower_components/moment-range/dist/moment-range',
-        json: 'vendor/json'
+        json: 'vendor/json',
+        bootstrap: '../bower_components/bootstrap-sass/assets/javascripts/bootstrap',
+        bootstrap_min: '../bower_components/bootstrap-sass/assets/javascripts/bootstrap.min'
```

Comment:
`bootstrap` is the JS for all bootstrap components, and `bootstrap.min` is the minified version of all bootstrap components - because we have `bootstrapCollapse` on line 9, you can remove these.